tap_tiktok_ads/__init__.py

```diff
@@ -16,7 +16,8 @@ def main():
     args = utils.parse_args(REQUIRED_CONFIG_KEYS)
 
     with TikTokClient(access_token=args.config['access_token'],
-                      user_agent=args.config['user_agent']) as tik_tok_client:
+                      user_agent=args.config['user_agent'],
+                      sandbox=args.config.get('sandbox', False)) as tik_tok_client:
```

Comment:
Why do we need to add this change in tap-tester PR? If it is necessary then do mention PR dependency.

Reply:
We have added this config param as we have a sandbox account and we need to pass `"sandbox": True`.

Reply:
For that we also have seprate card as well, right?

Reply:
Yes, it's TDL-18053.